tests/components/reolink/test_init.py

```diff
@@ -0,0 +1,183 @@
+"""Test the Reolink init."""
```

Comment:
This file can be greatly reduced by leveraging parameterizing the tests

Reply:
I don't understand what you mean by "parameterizing the tests", could you point to an example?